src/Raven.Server/ReplicationUtil/BaseReplicationLoader.cs

```diff
@@ -3,19 +3,20 @@
 using Raven.Abstractions.Logging;
 using Raven.Server.Documents;
 using Sparrow.Collections;
+using Sparrow.Logging;
 
 namespace Raven.Server.ReplicationUtil
 {
     public abstract class BaseReplicationLoader : IDisposable
     {
-        protected readonly ILog _log;
+        protected readonly Logger _logger;
         protected readonly DocumentDatabase _database;
         public readonly ConcurrentSet<BaseReplicationExecuter> Replications = new ConcurrentSet<BaseReplicationExecuter>();
 
         protected BaseReplicationLoader(DocumentDatabase database)
         {
             _database = database;
-            _log = LogManager.GetLogger(GetType());
+            _logger = _database.LoggerSetup.GetLogger<BaseReplicationLoader>(_database.Name);
```

Comment:
We want to use the actual type here.
Note that there is a non generic overload for this that can take a string, so you can use a GetType().FullNAme